code/2_analysis/scatterplots/fig2f.do

```diff
@@ -0,0 +1,24 @@
+* load across-crop productivity data
+use "data/intermediate/grace_merged/across_crop_avg_pot_yld.dta", clear
+
+* make scatterplot
+// calculate weighted mean of across-crop productivity by decile of delta_tws
+xtile dec_tws = delta_wd_cm_yr, nq(10) // creates deciles
+collapse (mean) py_z_wmean [aw=areahec], by(dec_tws) // collapse by decile
```

Comment:
@cheikhfall07 I did not realize we were taking area weighted potential yield averages - these are areas of planted area based on SAGE? Are other decile plots weighted? If so, we should update the text as I don't think it says this anywhere.

Reply:
Or is this just area weighted based on area of the GRACE cells? If so, we don't need to change text.

Reply:
The weights are just areas of GRACE cells. So, it is consistent with the text!